extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -1,199 +1,199 @@
 
-		$error = wfMsgExt('achievements-upload-not-allowed', array('parse'));
+		$error = wfMsgExt( 'achievements-upload-not-allowed', array( 'parse' ) );
 		return false;
 	}
 	return true;
 }
 
-function Ach_ArticleSaveComplete(	&$article, &$user, $text,
-					$summary, $minoredit, $watchthis,
-					$sectionanchor, &$flags, $revision,
-					&$status, $baseRevId) {
-	wfProfileIn(__METHOD__);
+function Ach_ArticleSaveComplete( &$article, &$user, $text,
+								  $summary, $minoredit, $watchthis,
+								  $sectionanchor, &$flags, $revision,
+								  &$status, $baseRevId ) {
+	wfProfileIn( __METHOD__ );
 
-	if($status instanceof Status && $status->ok == true && count($status->errors) == 0) {
+	if( $status instanceof Status && $status->ok == true && count( $status->errors ) == 0 ) {
 		// handle only successful edits and page creations
 
 		$awardingService = new AchAwardingService();
-		$awardingService->processSaveComplete($article, $user, $revision, $status);
+		$awardingService->processSaveComplete( $article, $user, $revision, $status );
 	}
 
-	wfProfileOut(__METHOD__);
+	wfProfileOut( __METHOD__ );
 	return true;
 }
 
-function Ach_GetHTMLAfterBody($skin, &$html) {
-	wfProfileIn(__METHOD__);
+function Ach_GetHTMLAfterBody( Skin $skin, &$html ) {
+	wfProfileIn( __METHOD__ );
 
 	global $wgOut, $wgTitle, $wgUser;
 
-	if($wgUser->isLoggedIn() && !($wgUser->getGlobalPreference( 'hidepersonalachievements' ))) {
-		if ($wgTitle->getNamespace() == NS_SPECIAL && array_shift(SpecialPageFactory::resolveAlias($wgTitle->getDBkey())) == 'MyHome') {
+	if( $wgUser->isLoggedIn() && !( $wgUser->getGlobalPreference( 'hidepersonalachievements' ) ) ) {
+		if( $skin->getTitle()->isSpecial( 'MyHome' ) ) {
 			$awardingService = new AchAwardingService();
-			$awardingService->awardCustomNotInTrackBadge($wgUser, BADGE_WELCOME);
+			$awardingService->awardCustomNotInTrackBadge( $wgUser, BADGE_WELCOME );
 		}
 
-		if((!empty($_SESSION['achievementsNewBadges']) || 5 == rand(1, 20)) && get_class(RequestContext::getMain()->getSkin()) != 'SkinMonobook') {
+		if( ( !empty( $_SESSION[ 'achievementsNewBadges' ] ) || 5 == rand( 1, 20 ) ) && get_class( RequestContext::getMain()->getSkin() ) != 'SkinMonobook' ) {
 			// this works only for Wikia and only in current varnish configuration
-			if (!headers_sent()) {
-				header('X-Pass-Cache-Control: no-store, private, no-cache, must-revalidate');
+			if( !headers_sent() ) {
+				header( 'X-Pass-Cache-Control: no-store, private, no-cache, must-revalidate' );
 			}
 
-			$notificationService = new AchNotificationService($wgUser);
-			$wgOut->addHTML($notificationService->getNotificationHTML());
-			if( isset($_SESSION['achievementsNewBadges']) )
-				unset($_SESSION['achievementsNewBadges']);
+			$notificationService = new AchNotificationService( $wgUser );
+			$wgOut->addHTML( $notificationService->getNotificationHTML() );
+			if( isset( $_SESSION[ 'achievementsNewBadges' ] ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)